### DISCUSSION TITLE
Bootstrap

### DISCUSSION THREAD

**Discussion by marcus-j-davies** [NRCHKB/nrchkb-ffmpeg-build#6] [General]
@Shaquu,

I am working on Alpine support.
This involves a shell script specific to its environment - given its many differences (apt -> apk, no sudo, some package name differences, etc etc)

However, it will still be one script that users need to use.
you can prob see how this works - Currently finding the best way to detect if running Alpine.

The shell bootstrap, is not fully able to detect correctly yet, but does work when it does.

https://github.com/NRCHKB/nrchkb-ffmpeg-build/blob/Alpine-Support/build-bootstrap.sh

any actions you are building, you can still just call into the specific shell script

**Comment by Shaquu** [comment]
I will try to test it tomorrow
